src/components/CurrentUser/CurrentUser.tsx

```diff
@@ -33,14 +39,17 @@ export const CurrentUser: React.FC<Props> = ({ userId, setSelectedUser }) => {
           </h3>
           <p className="CurrentUser__email">{user.email}</p>
           <p className="CurrentUser__phone">{user.phone}</p>
-          <button
-            data-cy="userButton"
-            className="CurrentUser__button"
-            type="button"
-            onClick={() => setSelectedUser(0)}
-          >
-            Clear
-          </button>
+          {userId && (
```

Comment:
It's a component with information about the user. Here user id always exists (you on props expected userId is always number)

Reply:
you need hide user button on todo instead 
<img width="827" alt="Screenshot 2022-05-24 at 11 19 41" src="https://user-images.githubusercontent.com/47606379/169984325-6fbfa1d3-17c7-42c2-95d2-16cc17ac913f.png">